.travis.yml

```diff
@@ -3,5 +3,8 @@ sudo: false
 node_js:
     - "0.12"
     - 4
+    - 6
+before_script:
+    - npm install typescript@^1.7.3
```

Comment:
Any way we can pull the version from package.json so we don't have to worry about keeping the versions in sync?